### PR TITLE
Marketplace UI update

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/appstore/general/app-store-overview.md [hunk 1]
```diff
@@ -17,24 +19,28 @@ This document describes the different sections of the Mendix Marketplace.
...
 | [Modules](/appstore/modules/) | Modules are packages of functionality for your app. They consist of a domain model, security options, and other elements like microflows, pages, or Java actions. Modules are mostly used for complex functionality that needs business or Java logic. |
+| **App Services** |  App services are packages of preconfigured capabilities, such as modules containing UI, logic, workflows, connectors, and APIs. They are designed to solve discrete business problems within the context of larger solutions.|
 | **App Design** | Start building your app based on a [layout](/refguide/layout) that is ready to use with commons structures and widgets or a [theme](/appstore/themes/) that contains styling options you can apply for different elements and devices. |
 | **Solutions** | Solutions are aimed at industries and domain problems. An out-of-the-box solution delivers instant value, and can be adapted using the core Mendix Platform to fully meet your requirements. |
+| **APIs** |  |
```
Comment (viralypatel) [inline content/appstore/general/app-store-overview.md:41]: "APIs" is a test category on accp env - will not show on production and can be removed.

Comment (Adam-Dupaski) [reply content/appstore/general/app-store-overview.md:41]: Thanks @viralypatel will remove.

**Comment by Adam-Dupaski** [comment]
@chandramohan-vellingiri it is much easier and more efficient if you comment on the specific line of the file - is that possible? That way I don't have to figure out where the feedback is directed. Thanks.

**Comment by Adam-Dupaski** [comment]
@chandramohan-vellingiri oh I see that's the case, don't know why it's not appearing here directly, sorry never mind.

**Comment by Adam-Dupaski** [comment]
@chandramohan-vellingiri I see what the issue is - I believe you are commenting on a commit rather than the current file (in Files). If you can leave your future feedback there, it'd be more efficient, thanks.

**Pull request merged**